### ISSUE TITLE
[RFC] Roadmap for GTNH

### ISSUE THREAD

**Issue by Dream-Master** [GTNewHorizons/GT-New-Horizons-Modpack#9884]
### Your Proposal

Pack development should be more focused on Themes for the next Update 

for example
* Pollution rework
* Botania Integration
* balancing Bees
* add new planetary stuff
* etc.

### Your Goal

Developer need to work together on the next Goal instead of adding random stuff 

* add random stuff is not bad but i think we much quicker with a straight goal for the next update

### Your Vision

Make the pack even better and more consistent.

**Comment by Dream-Master** [comment]
We can use https://github.com/orgs/GTNewHorizons/projects to coordinate stuff.
If somone have a better solution you can post it here.

**Comment by Houstonruss** [comment]
Yo no way GTNH:MEGA

**Comment by TwoTailedFox** [comment]
Can "Greater Biomes O'Plenty Integration" be added? TC doesn't have blocks registered from these mods for aspects (which could be added), and its crafted food doesn't stack.

**Comment by Chaoschaot232** [comment]
Another Goal should be the full updated for the overhauled version of Tainted Magic :3

**Comment by Dream-Master** [comment]
yes i add that already to the list

**Comment by MadRatSRP** [comment]
I want to add a small sugestion, i think it would be a great move if you also would target an entire quest chapter, for example:
2.1.3.0 - primarily focusing on all the things about Stone Age, rewatching quests and rewards, secondary on steam age. 
It doesnt mean that you wouldnt need to do stuff about other quest chapters, but focusing on entire quest chapters could make a big change i think

**Comment by Chaoschaot232** [comment]
I am actually using the 2.1.2.3qf Version and even there the Thaumonomicon Vanilla entries are shown instead of the actually ones (Caps and Wand).

Little reminder on that :)

**Comment by Dream-Master** [comment]
Yes the mushrooms spawns but I am not happy with it rn. I think all botania stuff need to get out of breeding crops or it has to spawn in other dims. And density need to changed as well.

**Comment by Chaoschaot232** [comment]
> Yes the mushrooms spawns but I am not happy with it rn. I think all botania stuff need to get out of breeding crops or it has to spawn in other dims. And density need to changed as well.

Tbh, what do you think about gating them in the Thaumonomicon and infuse Mushrooms with the color the Botania Mushroom should be and use in addition balanced shard and Terra from the Aspects 🤔

**Comment by lordclips** [comment]
IIRC _lots_ of people hate the idea of gating more things behind Thaum, so I doubt that would happen or really make sense

**Comment by Chaoschaot232** [comment]
Ok, then it's fair enough to not gate it behind Thaum when many things already are in it.

**Comment by Dream-Master** [comment]
> > Yes the mushrooms spawns but I am not happy with it rn. I think all botania stuff need to get out of breeding crops or it has to spawn in other dims. And density need to changed as well.
> 
> Tbh, what do you think about gating them in the Thaumonomicon and infuse Mushrooms with the color the Botania Mushroom should be and use in addition balanced shard and Terra from the Aspects 🤔

Better idea but botania is not a magic mod only

**Comment by Chaoschaot232** [comment]
> > > Yes the mushrooms spawns but I am not happy with it rn. I think all botania stuff need to get out of breeding crops or it has to spawn in other dims. And density need to changed as well.
> > 
> > 
> > Tbh, what do you think about gating them in the Thaumonomicon and infuse Mushrooms with the color the Botania Mushroom should be and use in addition balanced shard and Terra from the Aspects 🤔
> 
> Better idea but botania is not a magic mod only

Indeed it is always stated as Techmod with some magical touch. 🤔 I am then a bit lost or ... wait ... what about Witchery? Not sure where this is gated but maybe something could be added to the Brews or similar? Ok, it is after all a Magic mod but I would say that TC and even BloodMagic is much more Magic as Witchery since I used it in the past mainly for thhese brews which you need for other things.

**Comment by Chaoschaot232** [comment]
What also confuses me is why the are BoP's Poison Lakes are generated when this Liquid seems to have no use and seems also not to be storable in any kind of Tank?

BC Tank: Not storable (as one example)

I suggest to use it for negative potion effects or to crack it down in some higher tier GT Machines which are related to chemistry to give it a use. Otherwise Iw ould compleatly remove this useless fluid which nobody can use atm.

**Comment by Dream-Master** [comment]
> 

We can add Witchery recipes as well so why not use them too?  And we can gate botania stuff behind gt as well. Why not add a new electron tube to grow botania flowers in forestry farms. So the can’t grow normally they need a farm

**Comment by Dream-Master** [comment]
> What also confuses me is why the are BoP's Poison Lakes are generated when this Liquid seems to have no use and seems also not to be storable in any kind of Tank?
> 
> BC Tank: Not storable (as one example)
> 
> I suggest to use it for negative potion effects or to crack it down in some higher tier GT Machines which are related to chemistry to give it a use. Otherwise Iw ould compleatly remove this useless fluid which nobody can use atm.

Suggest recipes I am open for a use of this fluid

**Comment by Chaoschaot232** [comment]
> > 
> 
> We can add Witchery recipes as well so why not use them too? And we can gate botania stuff behind gt as well. Why not add a new electron tube to grow botania flowers in forestry farms. So the can’t grow normally they need a farm

But then don't forget that petals made out of them are plantable on every Grass Block and grow into Tall Botania Flowers which will give you mor petals back later on. With some groth support from Thaum (Lamp of Growth) they should grow faster.

Not sure if you'll let us use this functinality or not, it isn't so bad after all I think.

**Comment by MadRatSRP** [comment]
I want to add a suggestion which are in my mind will make the process of working on modpack much more effective:
When adding bugs, crashes, etc people can choose a theme or suggest new (its not required, optional), like for example if player wants to add a suggestion about reworking fighting mob/mobs, the group will be "Fighting mobs" and this group will contain things like changing the ways player can repair the armor, wrong mob tooltip in Waila, etc. This will help both the players and developers, players will  be able to see how the whole process of fighting mobs or something different is changing and developers could swap from theme to theme on which are they are working at the current time.

**Comment by shawnbyday** [comment]
It has been mentioned in Zeta chat about only working on non-bug fix/non-Botania item prior to Assembly Line and beyond. Personally, as a tester, I would like to see a development path that is not only organized and structured from a pack-vision perspective but also one that is conducive to us Zeta testers. So we can test as we do a playthrough and not bounce all over. It'll churn out as a more polished release if we can work through it this way.

I know Dream had also suggested that he wants to finish the Kevlar and Silicon lines as an exception to what was mentioned in Zeta.

I realize this entire pack exists because of volunteers and I'm extremely grateful for all of the work and effort because it's just the best experience modded Minecraft has to offer, in my maybe not so humble opinion. I'm not discouraging these volunteers to not develop new content to add to the pack, but simply wait until we are in the appropriate tier of review before it gets added. This again will help the polish via testing.

Below is obviously not a complete list, however, it is just a quick interpretation of some of the thing discussed. If you all want to try this route of development I would encourage us to all submit additional things to this list. We can denote which tier we will work on for each "release cycle" from here. I know we have more content now that goes past UHV but without being super deeply involved in it I'm not sure what it is since it has no quests I can see.

**Stone/Steam**
* Bug Fixes
* Botania Integration
* Quests

**LV**
* Bug Fixes
* Botania Integration
* Quests

**MV**
* Bug Fixes
* Botania Integration
* Quests

**HV**
* Bug Fixes
* Botania Integration
* Quests

**EV**
* Bug Fixes
* Botania Integration
* Quests

**IV**
* Bug Fixes
* Botania Integration
* Quests

**LuV**
* Quests

**ZPM**
* Quests

**UV**
* Quests

**UHV**
* Quests

**Beyond**
* Quests

### Future Implementation Ideas
This section would be for anything developers want to add or work on in the future that are pre-assembly line.
* Tier, Idea/Mod/etc.

**Comment by Ethryan** [comment]
Maybe someday implement
- #6640
Into GT5.
